File: ios/LookSeeProto/Models/Payment.swift
//
//  Payment.swift
//  LookSeeProto
//
//  Created by Christian Barbara on 11/13/25.
//
import Foundation
import SwiftUI

struct Payment: Hashable, Codable, Identifiable {
    var id: Int
    var cardProvider: String
    var cardNum: String
    var expireMonth: Int
    var expireYear: Int
    var cvv: Int
    
    var firstName: String
    var lastName: String
    var state: String
    var city: String
    var postCode: String
    var address1: String
    var address2: String
    var phone: String
    
}

File: Android/LookSee/app/src/main/java/looksee/angelll/com/models/Payment.kt
package looksee.angelll.com.models

/** Legacy billing-address/card record translated from Payment.swift. */
data class Payment(
    val id: Int? = null,
    val cardProvider: String,
    val cardNum: String,
    val expireMonth: Int,
    val expireYear: Int,
    val cvv: Int,
    val firstName: String,
    val lastName: String,
    val state: String,
    val city: String,
    val postCode: String,
    val address1: String,
    val address2: String? = null,
    val phone: String,
)
